build/azure-pipelines/darwin/product-build-darwin-test.yml

```diff
@@ -157,7 +157,7 @@ steps:
   - script: |
       set -e
       ./scripts/test.sh --build --tfs "Unit Tests"
-    displayName: Run unit tests (Electron)
+    displayName: Run core unit tests
```

Comment:
Remove changes to any of the product-build pipelines that don't start with sql, we don't use those. 

(and you're missing sql-product-build-darwin)

Reply:
Done.